src/App.js

```diff
@@ -1,13 +1,33 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+
+let testArray = [{task:'Fish', id: Date.now(), completed: false},{task:'More Fish', id: Date.now() + 1, completed: false}];
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
+  constructor (props) {
+    super();
+    this.state={todoList: testArray};
+  }
+
+  addToList = () => {
+    const todoListCopy = this.state.todoList;
+    
+    todoListCopy.push({
+      task: document.getElementById('todoInput').value , id: Date.now(), completed: false
```

Comment:
Curious, How did you come about this implementation? It's odd to see DOM manipulation like line 18 inside a React App.

Reply:
It's purely a case of it worked. 

I just talked to another student about it and had him suggest something. I was going to ask at the stand up about it tonight.